sema/commons/web/download_to_file.py

```
@@ -46,9 +46,9 @@ def save_web_content(
     with open(filepath, "w") as out:
         out.write(content)
     # remember what this file is for
-    os.setxattr(filepath, "user.web.url", url.encode("utf-8"))
-    os.setxattr(filepath, "user.web.mime_type", mime_type.encode("utf-8"))
-    os.setxattr(filepath, "user.web.profile", profile.encode("utf-8"))
+    os.setxattr(filepath, "user.web.url", url.encode("utf-8"))  # type: ignore
+    os.setxattr(filepath, "user.web.mime_type", mime_type.encode("utf-8"))  # type: ignore # noqa
```

Ruff

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)